tools/pipelines/templates/include-install.yml

```diff
@@ -26,4 +31,19 @@ steps:
       workingDirectory: ${{ parameters.buildDirectory }}
       script: |
         set -eu -o pipefail
+        registry="$(pnpm config get registry)"
```

Comment:
Rather than take this approach, it should be feasible to edit all of our .npmrc files and use the `npmAuthenticate` task. That might end up causing us to introduce an `.npmrc` location parameter into this install template. Or we might be able to get away with assuming it's at `${{ parameters.buildDirectory }}/.npmrc`. I can play around with this if the reviewer really hates the inlined auth logic. my experience with ADO feeds has not led me to much optimism in this area though.